bodo/tests/test_df_lib/test_jit_interop.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+
+import bodo
+import bodo.pandas as bd
+from bodo.tests.utils import _test_equal, pytest_mark_spawn_mode
+
+
+def test_bodo_df_to_jit():
+    df = pd.DataFrame(
+        {
+            "a": [1, 2, 3],
+            "b": [4, 5, 6],
+            "c": ["a", "b", "c"],
+        }
+    )
+    bdf = bd.from_pandas(df)
+
+    @bodo.jit()
```

Comment:
Set spawn=False explicitly to be safe?
```suggestion
    @bodo.jit(spawn=False)
```